src/components/pages/ipfs/Ipfs.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useRef, useEffect } from 'react';
```

Comment:
keep those changes somewhere but revert for now ( let's include that in a different PR just so we split up things )